MedscribeUI/src/components/Utilities/ThreeBtnSelector.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '@mantine/core';
+import { useState, useEffect } from 'react';
+
+interface ThreeBtnSelectorProps {
+  buttonLabelOptions: string[];
+  initialSelected?: string;
```

Comment:
Lets change the name of initialSelected to selectedPronoun

Reply:
btn selector is used for more than just pronouns so i used a generic name